code/processes/filealerter.q

```diff
@@ -7,6 +7,10 @@ polltime:@[value;`.fa.polltime;0D00:00:10]							// The period to poll the file
 alreadyprocessed:@[value;`.fa.alreadyprocessed;.proc.getconfigfile["filealerterprocessed"]]	// The location of the table on disk to store the information about files which have already been processed
 skipallonstart:@[value;`.fa.skipallonstart;0b]							// Whether to skip all actions when the file alerter process starts up (so only "new" files after the processes starts will be processed) 
 moveonfail:@[value;`.fa.moveonfail;0b]								// If the processing of a file fails (by any action) then whether to move it or not regardless
+decodepcaps:@[value;`.fa.decodepcaps;0b]							// Whether to parse PCAPs found by filealerter (causes connection to tp)
```

Comment:
I don't think this should be in here, and neither should the tickerplanttypes/tpconnssleepintv/tpcheckcycles
We don't need to build a start up dependency on the TP - we only need a connection to the TP when the file is parsed. 
So you should modify the sendtotickerplant function to take the tickerplanttype as a parameter, and create the connection at run time, and fail appropriately (e.g. print error) if it doesn't exist